lib/ansible/modules/crypto/openssl_publickey.py

```diff
@@ -211,10 +212,13 @@ def _check_privatekey():
             if not os.path.exists(self.privatekey_path):
                 return False
 
-            current_publickey = crypto.dump_publickey(
-                crypto.FILETYPE_ASN1,
-                crypto.load_publickey(crypto.FILETYPE_PEM, open(self.path, 'rb').read())
-            )
+            try:
+                current_publickey = crypto.dump_publickey(
+                    crypto.FILETYPE_ASN1,
+                    crypto.load_publickey(crypto.FILETYPE_PEM, open(self.path, 'rb').read())
+                )
+            except OpenSSL.crypto.Error:
```

Comment:
A simple `crypto.Error` doesn't work? That would also remove the doubled import from OpenSSL...